### PR TITLE
DOP-4943: Adding title to 404 page

### PR DESCRIPTION
**Pull request by biancalaube**
### Ticket

[DOP-4942](https://jira.mongodb.org/secure/RapidBoard.jspa?rapidView=953&projectKey=DOP&view=detail&selectedIssue=DOP-4943&quickFilter=26777#)

### Current Behavior
[current link](https://www.mongodb.com/docs/404/)

### Staged Behavior
[Staged link](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/docs-404/bianca.laube/main/)

### CODE REVIEW

**Review by i80and — APPROVED** [review]
It's good practice to have a trailing newline on text files. I recommend configuring your editor to ensure that.

**Review by i80and — APPROVED** [review]

File: snooty.toml [hunk 1]
```diff
@@ -1 +1,2 @@
 name = "docs-404"
+title = "404 page"
```
Comment (mmeigs) [inline snooty.toml]: Aha, so much better! Can we do something with a little more ✨ sparkle?
"404 - Not Found"
"MongoDB - 404"
"Not Found"
or whatever else floats your boat?

Comment (biancalaube) [reply snooty.toml]: just updated to "404 - Not Found"

**Review by mmeigs — APPROVED** [review]
❤️

**Pull request merged**